brewtils/rest/easy_client.py

```diff
@@ -1156,3 +1156,55 @@ def _check_chunked_file_validity(self, file_id):
             return True, metadata_json
         else:
             return False, metadata_json
+
+    @wrap_response(parse_method="parse_topic", parse_many=False, default_exc=FetchError)
+    def get_topic(self, topic_id):
+        """Get a topic
+
+        Args:
+            topic_id: Topic id
+
+        Returns:
+            The Topic
+
+        """
+        return self.client.get_topic(topic_id)
+
+    @wrap_response(parse_method="parse_topic", parse_many=True, default_exc=FetchError)
+    def get_topics(self):
+        """Get all Topics
+
+        Returns:
+            List[Topics]: List of Topics
+
+        """
+        return self.client.get_topics()
+
+    @wrap_response(parse_method="parse_topic", parse_many=False, default_exc=SaveError)
+    def create_topic(self, topic):
```

Comment:
There is a `patch_topic` in the client.py and no here. Take a look at `update_request` in this file for an example.